components/support/nimbus-fml/fixtures/fe/invalid/invalid_default_value_for_one_channel.fml.yaml

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+version: 1.0
+about:
+  description: The default value for `example-feature` on the `app-release` channel does not match its type.
+channels:
+  - app-debug
+  - app-release
+features:
+  example-feature:
+    description: An example feature
+    variables:
+      enabled:
+        description: Whether the feature is enabled
+        type: Boolean
+        default: false
+    defaults:
+      - value: {
+          "enabled": true
+        }
+        channel: app-debug
+      - value: {
+          "enabled": 1
+        }
```

Comment:
Inline dict literals are awkward in yaml. How about:

```yaml
- value:
    enabled: true
  channel: app-debug
- value:
    enabled: 1
```

Reply:
I'm fine with changing it, I've just noticed a pattern of us using the inline literals for default values in fml yml files